renata_paramastri.cfdg

```diff
@@ -0,0 +1,39 @@
+startshape human
```

Comment:
Really cool that you created a specific object you had in mind, I had a lot of trouble doing this, so I went with a more abstract piece in the end 👍.